### DISCUSSION TITLE
How can I contribute to this project?

### DISCUSSION THREAD

**Discussion by suzaku** [exaloop/codon#74] [General]
Good job! I've been waiting for something for Python that's like Crystal for Ruby.
I would like to help with such a project but I'm not familiar with C++ programming, what else could I contribute?

**Comment by arshajii** [comment]
Thanks for your interest! The main thing that comes to mind is the standard library ([`stdlib/`](https://github.com/exaloop/codon/tree/develop/stdlib) folder): there are currently a few modules that are either not present or missing a few functions. One example is `os` -- some of these *might* require calling the corresponding C functions via Codon's [`from C import`](https://docs.exaloop.io/codon/language/ffi). I'll also take another look and see if anything else sticks out...
